test/blackbox-tests/test-cases/alias-multiple.t

```diff
@@ -100,6 +99,5 @@ Even if the aliases list is empty
   2 |  (alias a)
   3 |  (aliases)
   4 |  (action (echo "I have run")))
-  Error: The 'alias' and 'aliases' fields are mutually exclusive. Please use
-  only the 'aliases' field.
+  Error: fields alias, aliases are mutually exclusive
```

Comment:
The quotes should be back on the field names

Reply:
I have another patch doing this. Do you want me to include it here?